Taarafo.Core/Migrations/20220910032910_UpdateEventModel.cs

```diff
@@ -1,4 +1,9 @@
-﻿using Microsoft.EntityFrameworkCore.Migrations;
+﻿// ---------------------------------------------------------------
```

Comment:
Good practice would be not to touch any other file that is not in the scope of the issue.